editions/tw5.com/tiddlers/concepts/ShadowTiddlers.tid

```diff
@@ -1,11 +1,19 @@
-title: ShadowTiddlers
-tags: Concepts
-modified: 201308091623
+\define actions()
+<$action-setfield $tiddler="$:/state/tab/moresidebar-1850697562" $field="text" $value="$:/core/ui/MoreSideBar/Shadows"/>
+<$action-setfield $tiddler="$:/state/tab/sidebar--595412856" $field="text" $value="$:/core/ui/SideBar/More"/>
+\end
+\define click(text)
+<$button actions=<<actions>>>$text$</$button>
+\end
 
 ShadowTiddlers are tiddlers that are loaded from within [[Plugins]]. Unlike ordinary tiddlers, they don't appear in most lists.
 
 ShadowTiddlers can be overridden with an ordinary tiddler of the same name. If that tiddler is subsequently deleted then the original shadow tiddler is automatically restored.
 
-The current shadow tiddlers are:
+!! Overridden Shadow Tiddlers
 
-<$list filter="[all[shadows]sort[title]]"/>
+<<list-links "[is[tiddler]is[shadow]sort[title]]">>
+
+!! Shadow Tiddlers
+
+Shadow tiddlers can be found in the <<click "Right Sidebar : More : Shadows tab">>
```

Comment:
Apologies, more consistency issues:

* The docs generally refer to the "sidebar" not the "right sidebar"
* The docs frequently refer to a specific sidebar tab without having a button to activate that tab. One might argue that this particular docs tiddler is enhanced by the button, but I think that consistency is more important. If we think that this is a worthwhile improvement then it needs to be implemented consistently across all the docs

I'd recommend removing the macros and changing the last line to:

```
A full list of shadow tiddlers can be found in the sidebar in the "More" -> "Shadows" tab
```